### PR TITLE
DFPL-1771: Flexserver configuration for preview environment for FPL

### PR DESCRIPTION
**Pull request by qasimnawaz-moj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DFPL-1771

### Change description ###

Flexserver configuration for preview environment for FPL

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: charts/fpl-case-service/values.preview.template.yaml [hunk 1]
```diff
@@ -268,9 +261,9 @@ ccd:
...
     extraEnvs:
       - name: DATA_STORE_URL
-        value: jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?ssl=disable&stringtype=unspecified
+        value: jdbc:postgresql://fpl-preview.postgres.database.azure.com:5432/{{ .Values.global.databaseNamePrefix }}data-store?sslmode=require&stringtype=unspecified
       - name: DB_URL
-        value: jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?ssl=disable&stringtype=unspecified
+        value: jdbc:postgresql://fpl-preview.postgres.database.azure.com:5432/{{ .Values.global.databaseNamePrefix }}data-store?sslmode=require&stringtype=unspecified
       - name: DB_USER
         value: hmcts
       - name: DB_PWD
```
Comment (cpareek) [inline charts/fpl-case-service/values.preview.template.yaml:269]: you will have to pass this `DB_PWD` as secret, this is the same secret you have setup on the server config on [cnp-flux-config](https://github.com/hmcts/cnp-flux-config)

**Review by cpareek — COMMENTED** [review]
you also have to update `ccd` chart version to the latest version https://github.com/hmcts/chart-ccd

File: charts/fpl-case-service/values.preview.template.yaml (hunk 1 above)
Comment (cpareek) [reply charts/fpl-case-service/values.preview.template.yaml:287]: this still need fixing?
see example here - https://github.com/hmcts/civil-ccd-definition/pull/3542/files#diff-42fed93979fdbd819455c6d1c5716828f1afd337e1c2980c925e1b9c3cb31e62L46

**Review by cpareek — APPROVED** [review]

**Review by ptrelease — DISMISSED** [review]

**Review by cpareek — APPROVED** [review]
LGTM

**Review by ptrelease — APPROVED** [review]

**Pull request merged**